io/net/virt-net/network_virtualization.py

```diff
@@ -145,6 +155,18 @@ def get_mcp_component(component):
                 return line.split()[-1].strip('{}\"')
         return ''
 
+    def is_lpar_in_server(self, server, lpar):
```

Comment:
This needs to be removed

Reply:
its already removed with latest update

Reply:
but it has to be removed via 1st commit, right ?

Reply:
how do we do that ? squash ?